bin/utils/prompt-command.js

```diff
@@ -113,10 +122,7 @@ module.exports = function promptForInstallation(packages, ...args) {
 							}
 
 							pathForCmd = path.resolve(process.cwd(), "node_modules", "@webpack-cli", packages);
-							if (packages === "serve") {
-								return require(pathForCmd).default.serve();
```

Comment:
you cannot remove `.serve()`

Reply:
The idea is that `.serve()` should just be the default function exported by the `serve` package, the way all the other packages export a default function. `.default.serve()` as it is in this file is not even correct, as `serve` currently does not have a default export. It's required and used correctly in the example: https://github.com/webpack/webpack-cli/blob/master/packages/serve/README.md#webpack-cli-serve

Reply:
sounds good, is there any other place the exported modules are used?

Reply:
I don't see any other places they are used. `prompt-command.js` should be the only place since it should make sure it is installed before trying to use a module.